src/axom/quest/tests/quest_point_in_cell_mfem.cpp

```diff
@@ -391,15 +394,21 @@ class PointInCellTest : public ::testing::Test
       m_mesh->GetNE(),
       constructTimer.elapsed()));
 
+    using SpacePtArray = axom::Array<SpacePt>;
+    using IndexArray = axom::Array<IndexType>;
+
     // Test that a fixed set of isoparametric coords on each cell
     // maps to the correct place.
-    axom::Array<SpacePt> isoPts = generateIsoParTestPoints(::TEST_GRID_RES);
+    SpacePtArray isoPts = generateIsoParTestPoints(::TEST_GRID_RES);
 
     const auto SZ = isoPts.size();
     axom::Array<SpacePt> spacePts(SZ, SZ);
     axom::Array<SpacePt> foundIso(SZ, SZ);
     axom::Array<IndexType> foundIDs(SZ, SZ);
 
+    axom::Array<SpacePt> foundIsoDevice(SZ, SZ, devAllocID);
+    axom::Array<IndexType> foundIDsDevice(SZ, SZ, devAllocID);
```

Comment:
Why not use type alias you created above?

Reply:
I originally added those type aliases to simplify switching between unified memory and host memory arrays, before realizing that unified memory didn't work :(

I'll just remove those aliases for consistency.